explorer/interpreter/resolve_unformed.h

```diff
@@ -0,0 +1,20 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef CARBON_EXPLORER_INTERPRETER_RESOLVE_UNFORMED_H_
+#define CARBON_EXPLORER_INTERPRETER_RESOLVE_UNFORMED_H_
+
+#include "explorer/ast/ast.h"
+#include "explorer/common/nonnull.h"
+
+namespace Carbon {
+
+// An introprocedural forward analysis that checks the may-be-formed states on
```

Comment:
Do you mean intra, not intro? (note, this also comes up in the cpp)

Reply:
Yes it should be intra.